src/System.Devices.Gpio/GpioController.cs

```diff
@@ -21,6 +21,10 @@ public GpioController(PinNumberingScheme numbering = PinNumberingScheme.Gpio)
             {
                 driver = new UnixDriver();
             }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                driver = new Windows10Driver();
+            }
             else
             {
                 throw new NotSupportedException($"Platform '{RuntimeInformation.OSDescription}' not supported");
```

Comment:
Is there a way to query whether the device is supported? e.g. `if (GpioController.IsSupported) { ...}`

Reply:
I would also like to look into how to find out more information about the platform (Windows version, Linux distro, etc.)